### ISSUE TITLE
Filters are not functioning

### ISSUE THREAD

**Issue by dlee13** [WiIIiam278/HuskChat#223]
My server is running HuskChat | v3.0.4

I tested with this advertising filter config:
```yaml
  ADVERTISING:
    type: filter
    enabled: true
    channels:
    - global
    - local
    private_messages: true
    broadcast_messages: false
```
and this channel:
```yaml
- id: local
  format: '&8[&r&#62dbfc&%server% &8&l| &r&eLocal&8]&f %fullname%&r&f:&e '
  broadcast_scope: LOCAL
  log_to_console: true
  restricted_servers: []
  filtered: true
  permissions: {}
  shortcut_commands:
  - /local
  - /l
```
![image](https://github.com/WiIIiam278/HuskChat/assets/4022777/85404bf6-2071-472d-bf62-3a202a3b4af6)

As far as I could tell, it seems like spam, repeat, and the emoji replacer are also not functioning.

**Comment by SwartZCoding** [comment]
Up this issue please @WiIIiam278 ! Tested for me too on Velocity side. HuskChat | v3.0.4
I confirm this one. All filters didn't work.

**Comment by WiIIiam278** [comment]
HuskChat's been a little lower on my priority list at them moments but yeah I will get to it as I do appreciative that this is a significant issue

**Comment by WiIIiam278** [comment]
Closed in #261
